raster/tiles/parser.py

```diff
@@ -408,7 +412,7 @@ def process_quadrant(self, indexrange, zoom):
             RasterTile.objects.bulk_create(batch)
 
         # Remove quadrant raster tempfile.
-        os.remove(dest_file_name)
+        # os.remove(dest_file_name)
```

Comment:
Can we leave this active? Is there a reason for commenting this out? If we don't need it, lets delete the line. Otherwise keep it in. The intention here was to save disk space while processing, which can be significant for large rasters. So if there is not a strong reason, lets keep this here.

Reply:
Hi - I've added that back in with a fix for the windows file locking. Thanks for spotting it